pom.xml

```diff
@@ -493,6 +493,13 @@
                     <configuration>
                         <source>8</source>
                         <target>8</target>
+                        <annotationProcessorPaths>
+                            <path>
+                                <groupId>org.projectlombok</groupId>
+                                <artifactId>lombok</artifactId>
```

Comment:
Is this related to supporting Java 24?

Reply:
Yeah without this we can't compile with Java 24.